package.json

```diff
@@ -143,7 +149,7 @@
     "@types/mocha": "^10.0.9",
     "@types/node": "^20.16.11",
     "@types/pixelmatch": "^5.2.6",
-    "@types/vscode": "^1.94.0",
+    "@types/vscode": "^1.90.0",
```

Comment:
Do you really need support for older vscode?

Reply:
Yes, becase your engines is 1.90.i take a look this problem. https://stackoverflow.com/questions/70137470/types-vscode-x-xx-greater-than-engines-vscode-x-xx. 
`"engines":{
    "vscode": "^1.90.0"
  },
`